### PR TITLE
renamed the yaml file name. change prefixes to 'not-jenkins' format.changed stress command to a string within a list.

### PR DESCRIPTION
**Pull request by harelassayag**
renamed the yaml file name. change prefixes to 'not-jenkins' format.changed stress command to a string within a list.

### CODE REVIEW

File: tests/enospc-30mins.yaml [hunk 1]
```diff
@@ -2,9 +2,9 @@ test_duration: 40
 n_db_nodes: 3
 n_loaders: 4
 n_monitor_nodes: 1
-user_prefix: 'gce-enospc-1-7'
+user_prefix: 'gce-enospc-not-jenkins'
```
Comment (roydahan) [inline tests/enospc-30mins.yaml]: change gce-  to cases-

File: tests/enospc-30mins.yaml [hunk 2]
```diff
@@ -2,9 +2,9 @@ test_duration: 40
...
 failure_post_behavior: destroy
-stress_cmd: cassandra-stress write no-warmup cl=QUORUM duration=20m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -errors ignore -pop seq=1..10000000
+stress_cmd: ["cassandra-stress write no-warmup cl=QUORUM duration=20m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -errors ignore -pop seq=1..10000000"]
```
Comment (roydahan) [inline tests/enospc-30mins.yaml]: Change back

File: tests/enospc-refresh-30mins.yaml [hunk 3]
```diff
@@ -6,8 +6,8 @@ tests: !mux
     enospc:
```
Comment (roydahan) [inline tests/enospc-refresh-30mins.yaml]: @amoskong, do we still use this file?
I think it's an old one and can be removed.

File: tests/refresh-30mins-120gb.yaml [hunk 4]
```diff
@@ -2,10 +2,10 @@ test_duration: 100
 n_db_nodes: 3
 n_loaders: 4
 n_monitor_nodes: 1
-user_prefix: 'gce-refresh-1-7'
+user_prefix: 'cases-refresh-120gb-not-jenkins'
 failure_post_behavior: destroy
 prepare_stress_cmd: cassandra-stress write no-warmup cl=QUORUM n=10000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -errors ignore -pop seq=1..10000000 -log interval=5
-stress_cmd: cassandra-stress mixed no-warmup cl=QUORUM duration=80m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -errors ignore -pop seq=1..10000000 -log interval=5
+stress_cmd: ["cassandra-stress mixed no-warmup cl=QUORUM duration=80m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -errors ignore -pop seq=1..10000000 -log interval=5"]
```
Comment (roydahan) [inline tests/refresh-30mins-120gb.yaml]: change back.

File: tests/enospc-refresh-30mins.yaml (hunk 3 above)
Comment (amoskong) [reply tests/enospc-refresh-30mins.yaml]: Yes, we can remove it.

File: tests/enospc-30mins.yaml (hunk 1 above)
Comment (amoskong) [reply tests/enospc-30mins.yaml]: how about `case-enospc-noJenkins`?

Comment (roydahan) [reply tests/enospc-30mins.yaml]: GCE doesn't allow capital letters in names.

**Comment by amoskong** [comment]
@harelassayag can you update the PR according Roy's comments?

**Comment by harelassayag** [comment]
@roydahan committed changes

**Comment by amoskong** [comment]
Backported this to branch-2.1

```
commit ac53b6c790cbcfbfdbe2f212c21016deeb0eac5f (HEAD -> branch-2.1, origin/branch-2.1)
Author: Harel Assayag <harel@scylladb.com>
Date:   Thu Nov 23 18:25:52 2017 +0200

    change gce to cases. remove list format brackts. deleted refresh-enospc file yaml'
    
    (cherry picked from commit 411823b76a54d02d8cc764101e5851d53d71ac77)

commit 7397405036d30d48c23cd7ffd902d1874fbe7a14
Author: Harel Assayag <harel@scylladb.com>
Date:   Tue Nov 21 16:09:26 2017 +0200

    renamed the yaml file name. change prefixs to 'not-jenkins' format.
    changed stress command to a string within a list.
    
    (cherry picked from commit bb6e80a3550ad9127e04f810ad80f2c49dd2e29a)
```

**Pull request merged**